src/intro/useCases/utils/getGreeting.ts

```diff
@@ -0,0 +1,3 @@
+export function getGreeting(hour: number): string {
+    return hour < 12 ? 'Good morning' : 'Hello';
```

Comment:
This implementation design isn't flexible because it will demand to change the function's signature when a new business rule comes in that will foster to consider minutes for example.

How about passing in the whole instance of a `Date`?
```diff
-  export function getGreeting(hour: number): string {
+  export function getGreeting(date: Date): string {
```